modules/_labs/button/react/lib/DeleteButton.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+import {ButtonSize} from './types';
+
+export interface DeleteButtonProps extends React.HTMLAttributes<HTMLButtonElement> {
```

Comment:
I decided to break up the different types of buttons into various components. The big reason for doing this was to  restrict prop combinations. It feels wrong to just render `null` when there's an invalid combination (or have incorrect styling). I would rather remove the possibility of invalid props from the API entirely.

Reply:
Why does `DeleteButton` not support icons or data labels?

Reply:
This was a decision made by design. I guess I could see it having a trash icon, but not a customizable one. The data label is really only meant for video/audio timestamps so I understand why it's not there.